content/en/docs/Writing policies/variables.md

```diff
@@ -146,6 +146,46 @@ busybox   0/1     Pending   0          25m   created-by=kubernetes-admin,run=bus
 In the output, we can clearly see the value of our `created-by` label is `kubernetes-admin` which, in this case, is the user who created the Pod.
 
 
+## Variables from container images
+
+Kyverno extracts image properties and make them available as variables. Whenever a new request has `containers` or `initContainers` defined, the image properties can be referenced as follows:
+
+Reference the image properties of container `my-container`:
+
+1. Reference the registry URL 
+
+`{{images.containers.\"my-container\".registry}}`
```

Comment:
I think it would be simpler to illustrate these built-in variables by using a container name which doesn't contain a dash and thereby removes the need to escape and double-quote that section.

Reply:
But probably add a note somewhere (or link to an existing note) so that users understand the syntax needed to escape if they have such a name.

Reply:
Updated!